index.html

```diff
@@ -131,6 +131,10 @@ <h2> events done by pydelhi </h2>
                     <ul>  
                         <li> bit.ly/pycon-2017 </li>
                     </ul>
+                    <li> PyDelhi Conf 2023 </li>
+                    <ul>
+                        <li> conference.pydelhi.org </li>
+                    </ul>
```

Comment:
Let's keep the remaining file as it is, we only need the latest Conference to be mentioned.